client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.tsx

```diff
@@ -43,6 +43,7 @@ export function getEnhancedTasks(
 	site: SiteDetails | null,
 	submit: NavigationControls[ 'submit' ],
 	displayGlobalStylesWarning: boolean,
+	globalStylesMinimumPlan?: string | null,
```

Comment:
Could this be non-nullable? Aren't we always going to have a minimum plan at this point?